src/go/rpk/pkg/cli/cmd/common/common.go

```diff
@@ -18,20 +19,27 @@ import (
 	"github.com/Shopify/sarama"
 	"github.com/burdiyan/kafkautil"
 	log "github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
 	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli/cmd/container/common"
 	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
 	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/kafka"
+	vtls "github.com/vectorizedio/redpanda/src/go/rpk/pkg/tls"
 )
 
 const FeedbackMsg = `We'd love to hear about your experience with redpanda:
 https://vectorized.io/feedback`
 
 const (
-	saslMechanismFlag  = "sasl-mechanism"
-	certFileFlag       = "tls-cert"
-	keyFileFlag        = "tls-key"
-	truststoreFileFlag = "tls-truststore"
+	saslMechanismFlag          = "sasl-mechanism"
+	enableTLSFlag              = "tls-enabled"
```

Comment:
I'm happy to keep these flags as is for now, but are we open to potentially renaming them down the line as an overall flag audit / redux? It may be valuable to add `-path` to the tls cert  / key / etc. flags. As well, since truststore is a java concept, there's not much of a reason to name the flag `truststore` when the file is being used in rpk (Go) as a PEM.